docs/guides/essentials/overview.md

```diff
@@ -1,2 +1,94 @@
 # Overview
 
+Daffodil consists of a few different layers, each which are appropriate for different use cases.
+
+## Driver
+The driver layer is the most powerful layer. It interacts with external platforms and structures data into Daffodil's specific model format. This enables a single frontend application to interface with multiple platforms without dealing with the differences or intricacies of each of those platforms.
+
+Driver usage is straightforward; simply call a method on the driver class and use the result! Nearly all driver methods return an [`Observable`](https://rxjs.dev/api/index/class/Observable) to encapsulate the async behavior of network requests.
+
+```ts
+import {DaffProductServiceInterface, DaffProductDriver} from '@daffodil/product/driver';
+
+@Component({
+	template: `
+	  <div>{{ (product$ | async).name }}</div>
+	`
+})
+class AppComponent implements OnInit {
+	product$: Observable<DaffProduct>;
+
+	constructor(
+		@Inject(DaffProductDriver) private productDriver: DaffProductServiceInterface
+	) {}
+
+	ngOnInit() {
+		this.product$ = this.productDriver.get('productId')
+	}
+}
+```
+
+See [drivers](/docs/guides/essentials/drivers.md) for more information.
+
+## State
+The state layer builds upon the driver layer to provide app-wide redux state. Driver operations are triggered by dispatching actions and results (or the errors) are selectable from state. Daffodil abstracts both of these features in *facades*.
+
+State usage is more complex than driver but offers more advanced capabilities like loading states and error handling.
+
+```ts
```

Comment:
1. Two different examples, so they should probably be rendered in two separate code blocks?
2. Same issue with  linting in this example
3. Need to add missing imports (missing `input`,`DaffProduct`, etc)